index.d.ts

```diff
@@ -9,6 +9,8 @@ export interface SwaggerPageOptions {
     definitions?: {[key: string]: any};
     routePrefix?: string;
     forceSecure?: boolean;
+    host?: string;
+    schemes: string[];
```

Comment:
You should replace the line above with `schemes?: string[];` (note the question mark), this is the reason why the compilation fails during the Travis test.